contracts/src/libraries/EigenDAHasher.sol

```diff
@@ -117,6 +117,17 @@ library EigenDAHasher {
      * @param blobHeader the V2 blob header to hash
      */
     function hashBlobHeaderV2(BlobHeaderV2 memory blobHeader) internal pure returns(bytes32) {
```

Comment:
Do we ever have to hash BlobCertificate from chain? 
Wondering how this affects hasher of BlobCertificate if it exists